src/sentry/api/endpoints/group_tombstone_details.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry.api.bases import ProjectEndpoint
+
+from sentry.models import (
+    GroupHash, GroupTombstone,
+)
+
+
+class GroupTombstoneDetailsEndpoint(ProjectEndpoint):
+
+    def delete(self, request, project, tombstone_id):
+        """
+        Remove a GroupTombstone
+        ```````````````
+
+        Undiscards a group such that new events in that group will be captured.
+        This does not restore any previous data.
+
+        :pparam string organization_slug: the slug of the organization.
+        :pparam string project_slug: the slug of the project to which this tombstone belongs.
+        :pparam string tombstone_id: the ID of the tombstone to remove.
+        :auth: required
+        """
+        GroupHash.objects.filter(
```

Comment:
we should probably change this endpoint so that it returns a 404 if we can't find a tombstone (you can use `ResourceDoesNotExist`)